codespell_lib/data/dictionary.txt

```diff
@@ -10551,38 +10571,81 @@ classe->class, classes,
 classees->classes
 classess->classes
 classesss->classes
+classication->classification
+classications->classifications
 classied->classified
 classifcation->classification
+classifcations->classifications
 classifed->classified
 classifer->classifier
 classifers->classifiers
+classifes->classifies
 classificaion->classification
+classificaions->classifications
+classifiication->classification
+classifiications->classifications
+classifiied->classified
+classifiier->classifier
+classifiiers->classifiers
+classifiies->classifies
 classifiy->classify
 classifiying->classifying
 classrom->classroom
 classroms->classrooms
 classs->class
 classses->classes
+classsic->classic
+classsical->classical
+classsically->classically
+classsification->classification
+classsifications->classifications
+classsified->classified
+classsifier->classifier
+classsifiers->classifiers
+classsifies->classifies
+classsify->classify
+classsifying->classifying
+classsroom->classroom
+classsrooms->classrooms
 clatified->clarified
 claus->clause
 clcoksource->clocksource
 clcosed->closed
-clea->clean
+clea->clean, clear,
```

Comment:
Perhaps `cleat` or `clew`, or even `plea`, although such typos must be rather rare.

Reply:
I will add `cleat`. `clew` and `plea` are too rare.